snuba/settings/settings_self_hosted.py

```diff
@@ -16,4 +17,7 @@
 DOGSTATSD_HOST = env("DOGSTATSD_HOST")
 DOGSTATSD_PORT = env("DOGSTATSD_PORT")
 
+# Dataset readiness states supported in this environment
+SUPPORTED_STATES: List[str] = ["complete"]
```

Comment:
I think it's better to leave this the same as the other environments for now.

If we make this setting different at first, then this will change for ST _implicitly_ when we deploy the backend support for the setting.

I think it's better to deploy the support on the backend first, and the _explicitly_ change the setting to restrict things.

Reply:
That makes sense to me. Updated self-hosted settings to support all readiness states for now (until support for backend is done).